package.json

```diff
@@ -20,10 +20,11 @@
     "testharness/"
   ],
   "scripts": {
-    "test": "mocha",
+    "test": "node test/runner.js",
     "lint": "eslint lib bin",
-    "prepare": "npm run copy-testharness",
-    "copy-testharness": "copyfiles -u 2 wpt/resources/testharness.js testharness/"
+    "prepare": "npm run copy-testharness && npm run copy-resources",
+    "copy-testharness": "copyfiles -u 2 wpt/resources/testharness.js testharness/",
+    "copy-resources": "copyfiles -u 2 wpt/resources/testharness.js test/tests/resources/"
```

Comment:
This isn't needed for preparing the published package, so it seems like this should be a pretest hook, not part of prepare.